src/bundle/Resources/config/criteria.yaml

```diff
@@ -0,0 +1,40 @@
+services:
+    Ibexa\Rest\Server\Input\Parser\ContentType\Criterion\CriterionProcessor:
+        parent: Ibexa\Contracts\Rest\Input\Parser\Query\Criterion\BaseCriterionProcessor
+
+    Ibexa\Rest\Server\Input\Parser\ContentType\SortClause\SortClauseProcessor:
+        parent: Ibexa\Contracts\Rest\Input\Parser\Query\SortClause\BaseSortClauseProcessor
+
+    _instanceof:
+        Ibexa\Rest\Server\Input\Parser\ContentType\Criterion\ContentTypeCriterionInterface:
+            tags:
+                - 'ibexa.rest.content_type.criterion'
```

Comment:
This should be placed on top of the file, directly under `services` key - as this affects all services in this file.

Also note that this does not mean `ContentTypeCriterionInterface` parsers will be automatically tagged. It only means that services declared **in this file** that are instances of that will be tagged. You need `Container::registerForAutoconfiguration()` in extension instead for this to happen.